chain-core/src/common/fixed.rs

```diff
@@ -25,45 +26,196 @@ fn continued_fraction(
 }
 
 // https://en.wikipedia.org/wiki/Exponential_function#Continued_fractions_for_ex
-fn exp_series(
+fn exp2_series(
     x: FixedNumber,
+    y: FixedNumber,
 ) -> (
     impl Iterator<Item = FixedNumber>,
     impl Iterator<Item = FixedNumber>,
 ) {
-    let it_a = vec![FixedNumber::from_num(1), FixedNumber::from_num(1)]
+    let x2 = x * x;
+    let it_a = vec![FixedNumber::from_num(1), y * 2 - x]
         .into_iter()
-        .chain((2..).map(move |n| x + FixedNumber::from_num(n)));
-    let it_b = iter::once(1).chain((1..).map(|n| -n)).map(move |n| x * n);
+        .chain((0..).map(move |i| y * (i * 4 + 6)));
+    let it_b = iter::once(2 * x).chain(iter::repeat(x2));
     (it_a, it_b)
 }
 
 /// Compute exponencial function with continued fraction method
```

Comment:
perhaps expand a comment to clarify that this is computing `exp(x/y)`

Reply:
Done